tools/update_chaldea_data.py

```
@@ -24,9 +24,9 @@
 
 ATLAS_API = "https://api.atlasacademy.io"
 
-# 输出目录：assets/resource/Chaldea/
+# 输出目录：agent/utils/Chaldea/
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)